extension/portforward/portforward.js

```diff
@@ -159,6 +162,7 @@ class PortForward {
       }
       
       log.info("PORTMAP: Add",map);
+      await (shieldManager.addIncomingRule(map.protocol, map.toIP, map.dport));
```

Comment:
no need to add bracket for native await

Reply:
Seems native await is not allowed in 'async' block from 'async/await' library